internal-client/src/main/java/com/tencent/rss/client/impl/grpc/ShuffleServerGrpcClient.java

```diff
@@ -393,7 +394,7 @@ public RssGetShuffleResultResponse getShuffleResult(RssGetShuffleResultRequest r
           response = new RssGetShuffleResultResponse(ResponseStatusCode.SUCCESS,
               rpcResponse.getSerializedBitmap().toByteArray());
         } catch (Exception e) {
-          throw new RuntimeException(e);
+          throw new RssException(e);
```

Comment:
It's not caused by RSS, shouldn't be changed

Reply:
OK

Reply:
> It's not caused by RSS, shouldn't be changed

why is this not caused by RSS? I think this exception is caused by shuffle server's heavy load. The reader cannot read data successfully.

Reply:
The grpc call is success, so it shouldn't be marked as RSS related

Reply:
OK, I understand.